src/main/java/com/SBS/Controllers/MatchController.java

```diff
@@ -0,0 +1,33 @@
+package com.SBS.Controllers;
+
+import com.SBS.Models.Match;
+import com.SBS.Models.MatchResult;
+import com.SBS.Services.MatchService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/match")
+public class MatchController {
+    MatchService matchService;
+
+    public MatchController(MatchService matchService) {
+        this.matchService = matchService;
+    }
+
+    @GetMapping
```

Comment:
Na pewno Lista? Jak bedziesz mial duzo meczy to moze lepiej to pagami zwracać? To już jest pytanie biznesowe czy do Ciebie jak chcesz to wykorzystac.

Reply:
proponuję zostawić to jako zadanie przy wprowadzaniu CRUD Repository , gdyż tam jest zdefiniowana metoda repo.save(pageable)